libraries/chain/CMakeLists.txt

```diff
@@ -131,6 +131,7 @@ target_link_libraries( eosio_chain PUBLIC fc chainbase Logging IR WAST WASM Runt
 target_include_directories( eosio_chain
                             PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include" "${CMAKE_CURRENT_BINARY_DIR}/include"
                                    "${CMAKE_CURRENT_SOURCE_DIR}/../wasm-jit/Include"
+                                   "${CMAKE_CURRENT_SOURCE_DIR}/../rapidjson/include"
```

Comment:
This really needs to be an INTERFACE library

Reply:
Can you provide an example of what that would look like. I didn't see an example in our code.

Reply:
something like (I'm just winging this so may not be fully correct)
```
add_library(rapidjson INTERFACE)
target_include_directories(rapidjson INTERFACE rapidjson/include)
target_compile_definitions(rapidjson INTERFACE -DRAPIDJSON_NAMESPACE=eosio_rapidjson)
```

then wherever you want to use it just
```
target_link_libraries(eosio_chain rapidjson)
```

Reply:
I'm not sure if making the RAPIDJSON_NAMESPACE thing part of the interface is a good idea or not though.